custom_components/opnsense/coordinator.py

```
@@ -51,58 +52,44 @@ async def inner(self, *args, **kwargs):
+                if lease["online"] == "online":
+                    lease_stats["online"] += 1
+                if lease["online"] == "offline":
+                    lease_stats["offline"] += 1
+        return lease_stats
 
     @_log_timing
-    async def _get_arp_table(self):
-        return await self._client.get_arp_table(True)
-
-    async def _async_update_data(self):
+    async def _async_update_data(self) -> Mapping[str, Any]:
```

Ruff

`_async_update_data` is too complex (15 > 10)

(C901)